src/assets/styles/base.scss

```diff
@@ -56,7 +56,7 @@ h1, h2, h3, h4 {
 
 h1, h2{
   letter-spacing: -0.04rem;
-  font-weight: 700;
+  font-weight: bold;
```

Comment:
This project only has 2 font weights: 400 (normal) and 700 (bold). Anything else may or may not work as expected.